deploy/static/05-starboard-operator.config.yaml

```diff
@@ -211,9 +211,3 @@ kind: Secret
 metadata:
   name: starboard
   namespace: starboard-operator
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: starboard-conftest-config
```

Comment:
Static YAMLs hardcode config audit plugin to Polaris so we won't create ConfigMap for Conftest